cmd/celestia-appd/cmd/modify_root_command_multiplexer.go

```diff
@@ -49,22 +49,22 @@ func modifyRootCommand(rootCommand *cobra.Command) {
 		panic(err)
 	}
 
-	var extraArgs []string
+	var v3Args []string
 	if v2UpgradeHeight != "" {
-		extraArgs = append(extraArgs, "--v2-upgrade-height="+v2UpgradeHeight)
+		v3Args = append(defaultArgs, "--v2-upgrade-height="+v2UpgradeHeight)
```

Comment:
I think this is the correct way to do it. Otherwise, if `v2UpgradeHeight == ""`, then `v3Args` will be empty
```suggestion
	v3Args := defaultArgs
if v2UpgradeHeight != "" {
		v3Args = append(v3Args, "--v2-upgrade-height="+v2UpgradeHeight)
```

Reply:
True, good catch